spec/views/users/description/champs/_render_list_champs.html.haml_spec.rb

```diff
@@ -33,9 +33,8 @@
       render 'users/description/champs/render_list_champs.html.haml', champs: Champ.all, order_place: 0
     end
 
-    it 'should render a number input with the right data-attribute' do
-      expect(view).to render_template(partial: 'users/description/champs/_dossier_link',
-                                      locals: { champ: champ })
+    it 'should render an input for the dossier link' do
+      expect(rendered).to have_css("input[type=number][placeholder=#{champ.libelle}]")
```

Comment:
Je comprends, et j'avoue que du coup j'hésite.

Moi ce qui me genait dans le test initial c'est que j'avais l'impression qu'on testait littéralement le code, à la limite du `expect(function).to be_called`.

Et je voulais aussi qu'on ait ici un test qui soit similaire aux autres dans le fichier.

Au final, le nouveau test permet de tester à la fois ce que tu testais, en plus du test qui pourrait se trouver dans `_dossier_link.html.haml_spec`.

@mmagn t'en penses quoi ?